js/script.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+	var searchResults = [];
+	var counter = 0;
+	var intervalId = 0;
+	$("#submit").on("click", function(e){ 
+		e.preventDefault();
+		
+		var webSite = "https://www.reddit.com/search.json?q="+$("#input").val()+"+nsfw:no";
+		
+		$.get(webSite).done(function(data) {
+			searchResults = data;
+
+			var children = searchResults.data.children;
+
+			var hasUrl = function(object){
+				if(object.data.url) {
+					return true;
+				}	else {
+					return false;
+
+				}
```

Comment:
`data.url` works great if the post is a direct link to an image, but in some cases it can return a link to a text post or website. Look at `data.thumbnail` or `data.preview.images[0].source.url` for an image link.